src/coreclr/jit/emitarm64.cpp

```diff
@@ -9311,6 +9317,17 @@ void emitter::emitIns_R_R(instruction     ins,
             fmt = IF_SVE_BI_2A;
             break;
 
+        case INS_sve_fmov:
+            assert(insOptsScalableAtLeastHalf(opt));
+            assert(isVectorRegister(reg1));                        // ddddd
+            assert(isPredicateRegister(reg2));                     // gggg
+            assert(isValidVectorElemsize(optGetSveElemsize(opt))); // xx
+            fmt = IF_SVE_BV_2B;
+
+            // MOV is an alias for FMOV, and is always the preferred disassembly.
+            ins = INS_sve_mov;
```

Comment:
This is odd. The docs suggest it should show fmov: https://docsmirror.github.io/A64/2023-09/fmov_cpy_z_p_i.html

Reply:
Unhelpfully, that's only on the CPY page in the docs, not FMOV. A comment would be great.

Reply:
Sure thing; fixed.